main.js

```diff
@@ -0,0 +1,234 @@
+    } else if (playerTurn === "playerO") {
+        cellLookup[cellID].takenO = true; //sets taken status to true
+        document.getElementById(cellLookup[cellID].ID).innerHTML = "O";
+        document.getElementById('statusBoard').innerHTML = ("Player O took cell " + cellLookup[cellID].ID);
+        playerOList.push(cellLookup[cellID].ID)
+        winCheck(playerOList);
+        playerTurn = "playerX" //switches player turn to other player
+        console.log("inside else if " + playerTurn)
+        document.getElementById('whoseTurn').innerHTML = "It's " + playerTurn + "'s turn."
+        console.log("Player X moves: " + playerXList)
+        console.log("Player O moves: " + playerOList)
+    } else {
+        cellLookup[cellID].takenX = true;
+        document.getElementById(cellLookup[cellID].ID).innerHTML = "X";
+        playerXList.push(cellLookup[cellID].ID) //adds player move to player move list
+        document.getElementById('statusBoard').innerHTML = ("Player X took cell " + cellLookup[cellID].ID);
+        winCheck(playerXList)
+        playerTurn = "playerO" //switches player turn to other player
+        console.log("inside else for " + playerTurn)
+        document.getElementById('whoseTurn').innerHTML = "It's " + playerTurn + "'s turn."
+        console.log("Player X moves: " + playerXList)
+        console.log("Player O moves: " + playerOList)
+    }
+}
+
+//----------------------event listeners for cell move function ------------//
+document.getElementById('cell-0').addEventListener('click', () => {
+    cellMove('cellZero');
+}, 2000)
+document.getElementById('cell-1').addEventListener('click', () => {
+    cellMove('cellOne');
+}, 2000)
+document.getElementById('cell-2').addEventListener('click', () => {
+    cellMove('cellTwo');
+}, 2000)
+document.getElementById('cell-3').addEventListener('click', () => {
+    cellMove('cellThree');
+}, 2000)
+document.getElementById('cell-4').addEventListener('click', () => {
+    cellMove('cellFour');
+}, 2000)
+document.getElementById('cell-5').addEventListener('click', () => {
+    cellMove('cellFive');
+}, 2000)
+document.getElementById('cell-6').addEventListener('click', () => {
+    cellMove('cellSix');
+}, 2000)
+document.getElementById('cell-7').addEventListener('click', () => {
+    cellMove('cellSeven');
+}, 2000)
+document.getElementById('cell-8').addEventListener('click', () => {
+    cellMove('cellEight');
+}, 2000)
+
+
+//------------------------start button event listener --------------------//
+
+//one-player button 
+document.getElementById('one-player').addEventListener('click', () => {
+    onePlayerInit();
+});
+function onePlayerInit() {
+    document.getElementById('statusBoard').innerHTML = "The computer is ready. You are X, computer is O - X goes first.";
+    setInterval(gameTimer, 1000)
+}
+
+//two-player button
+document.getElementById('two-player').addEventListener('click', () => {
+    twoPlayerInit();
```

Comment:
instead of wrapping your callback in an anonymous arrow function (which does work just fine) you can pass the function name in directly without calling it, and it should work the same way e.g. `element.addEventListener('click', twoPlayerInit)`